mailpoet/assets/js/src/automation/editor/components/inserter-listbox/listbox_group.tsx

```diff
@@ -0,0 +1,40 @@
+import { HTMLAttributes } from 'react';
+import { speak } from '@wordpress/a11y';
```

Comment:
regarding a11y. What I noticed, I couldn't navigate to the Workflow itself using `TAB` :thinking:

Reply:
It seems to work for me. TAB to get to the workflow, then arrow up/down to move within:

https://user-images.githubusercontent.com/141436/169025222-ce7e10bd-93e0-4e0a-8053-6ed4fd2fe073.mov

Reply:
found the issue. seems firefox does not like the css property https://github.com/mailpoet/mailpoet/pull/4131/files#r875785786

Reply:
@websupporter Good catch! Makes sense, as I forgot the vendor-specific property there. Does [this](https://css-tricks.com/copy-the-browsers-native-focus-styles/) work for you or should I use just a fixed color? 🤔

Reply:
That should work :) :bow:

Reply:
Or maybe we should do what WP does:

<img width="1117" alt="Screen Shot 2022-05-18 at 13 45 20" src="https://user-images.githubusercontent.com/141436/169031436-b3d32423-84dc-4fb3-89d3-35d674ed3a4f.png">

Not sure if it's safe to use those variables directly though...

Reply:
When I see this correctly, we have those variables, because we enqueue 'wp-edit-post' and the colors get overwritten, when I change the styles in my profile. I would like to see this for consistency and would think those variables fall under the "backwards compatibility" policy of WP.

Reply:
Addressed in https://github.com/mailpoet/mailpoet/pull/4131/commits/e4c2da9ca1033204a142eb13c6fd0a214c9436a0.